test/sample_policies/always_violate/policies/templates/gcp_always_violates_v1.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-always-violates-v1
+spec:
+  crd:
+    spec:
+      names:
+        kind: GCPAlwaysViolatesConstraintV1
+        plural: gcpalwaysviolatesconstraintsv1
+      validation:
+        openAPIV3Schema:
+          properties: {}
+  targets:
+   validation.gcp.forsetisecurity.org:
```

Comment:
can we use the new form of targets here?
example: https://github.com/forseti-security/config-validator/blob/10863ea54f95e93d392fd7849f7148ec668b6197/pkg/gcv/configs/config_test.go#L220

We're trying to retire this old format.

Reply:
I don't see `- target: validation.gcp.forsetisecurity.org` in the latest snapshot?

Reply:
Instead of manually updating this, could we simply use a subtree of the policy library?

Reply:
Yes and no. I would prefer having a clean set of tests that does not depends on policy library, and then another set of tests that depends on it. So this pull request falls into the former.

Reply:
> I don't see - target: validation.gcp.forsetisecurity.org in the latest snapshot?

Sorry I was trying to put comments but marked the conversation resolved. I am not able to understand the new syntax. There seems to be an YAML syntax issue there in the following snippet, in the third line (or did I miss something)?

```
  targets:
    - target:validation.gcp.forsetisecurity.org
      rego: |
            # Some random
            # rego code
```

Reply:
Resolved. It looks like the referenced code in https://github.com/forseti-security/config-validator/blob/master/pkg/gcv/configs/config_test.go missed a space between `target:` and `validation.gcp.forsetisecurity.org`.

Reply:
I opened https://github.com/forseti-security/config-validator/pull/43 to fix the yaml syntax error in config-validator.